CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   1. False positive when the same loop variable is used in two loops in sequence.
   2. False negative when the loop variable can be simplified, but is also shadowed in the
      the loop body.
+- Fix HTML report to link correctly to speficif errors.
```

Comment:
There's a typo on "specific". Also, please mention "on the PythonTA documentation website" (otherwise it isn't clear where this is linking to).